src/tree.rs

```diff
@@ -231,6 +231,8 @@ impl Tree {
 
     /// Perform a multi-key serializable transaction.
     ///
+    /// User-provided function `f` may be run multiple times in case of conflicts.
+    ///
```

Comment:
please get rid of all of the other comments except this one, which is the only one we discussed earlier.